modules/retail/pages/retail-network-arch.adoc

```diff
@@ -16,8 +16,16 @@ We recommend you have a fast network connection between the branch server and it
 Branch servers provide services for PXE boot, and act as an image cache, Salt broker, and proxy for software components (RPM packages).
 The branch server can also manage local networking, and provide DHCP and DNS services.
 
-{smr} Branch servers are implemented as enhanced {susemgr} Proxy servers.
-For technical background information on {smr} Branch servers, see also the documentation on {susemgr} Proxy servers (xref:installation:install-proxy-unified.adoc[]).
+
+ifeval::[{suma-content} == true]
+{smr} Branch Servers are implemented as enhanced {productname} Proxies.
+For technical background information on {smr} Branch Servers, see also the documentation on {productname} Proxy servers (xref:installation:install-proxy-unified.adoc[]).
+endif::[]
+
+ifeval::[{uyuni-content} == true]
+{smr} Branch Servers are implemented as enhanced {productname} Proxies.
+For technical background information on {smr} Branch Servers, see also the documentation on {productname} Proxy servers (xref:installation:install-proxy-uyuni.adoc[]).
+endif::[]
```

Comment:
Can we do these without the parenths? Use the standard "for more ..., see..."

Reply:
done. we will have to change other retail files accordingly :)